tests/e2e/specs/dashboard/dashboard.js

```diff
@@ -30,6 +30,8 @@ describe('Stories Dashboard', () => {
 
     await expect(page).toMatch('My Stories');
 
-    await percySnapshot(page, 'Stories Dashboard');
+    await percySnapshot(page, 'Stories Dashboard', {
+      percyCSS: `.dashboard-grid-item-date { display: none; }`,
```

Comment:
In #4409 I proposed mocking dates instead. Did you explore that? Not worth the hassle for this one occasion?

Reply:
I looked at it yes. i forgot to put it in the PR description. It seems the issue is because of the order in which tests are run. It is doesn't always seem to follow. So sometimes, the dashboard runs in order, meaning that site is 1 minute ago, sometimes, it is was 3 minutes ago becuase a test runs later. I looked into an MU plugin and changing system time in docker container. Both felt super hacky and like we were making these tests run in a completely artificial environment. Hiding these lines isn't great, but it much cleaner than other alternatives.